tests/wpt/mozilla/meta/mozilla/interfaces.html.ini

```diff
@@ -0,0 +1,4 @@
+[interfaces.html]
+  [Interfaces exposed on the window]
+    expected: FAIL
```

Comment:
We need to update these files to include the newly exposed interfaces instead. We should not be adding expected failures for the interfaces tests.